src/components/HomepageFeatures/index.js

```diff
@@ -1,64 +1,142 @@
+// REFER TO THIS SECTION TO CONTRIBUTE TO THIS FILE
+
+// Here 
+
+// 1. hp refers to Homepage
+// 2. cont refers to Container
+// 3. p refers to Paragraph
+
+// For Example :-
+
+// (hp_cont_3_btn2_content) refers to "Content present in Button number 2  of Container number 3 on Homepage"
+
+// File for homepage
+
```

Comment:
FIrstly don't use unnecessary spaces and comments 
- comments for importing is of no use 
- please consise the length of comment. Here is what I thing should be there 

```
//The naming convention used in this file is mentioned below.
// 1. hp refers to Homepage
// 2. cont refers to Container
// 3. p refers to Paragraph
// (hp_cont_3_btn2_content) refers to "Content present in Button number 2  of Container number 3 on Homepage"
```